code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -14,16 +14,20 @@ DISREGUARD THIS FILE IF YOU'RE INTENDING TO CHANGE ASPECTS OF PLAYER CONTROLLED
 	attack_verb = "slash"
 	attack_sound = 'sound/weapons/slice.ogg'
 	miss_sound = 'sound/weapons/slashmiss.ogg'
-	burnmod = 1.5 //Ow my plant
-	heatmod = 1.5 //Ow my plant
+	burnmod = 2
```

Comment:
🤔

Reply:
bringing the unused file into parity with the used phytosian file